### PR TITLE
WIP: Adding a sample job to use hive role to claim clusters

### PR DESCRIPTION
**Pull request by frenzyfriday**
As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running
- [ ] Appropriate documentation exists and/or is up-to-date:
  - [ ] README in the role
  - [ ] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci/playbooks/hive/claim.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- name: Install deps and prepare for KUTTL run
+  hosts: ""
```
Comment (frenzyfriday) [inline ci/playbooks/hive/claim.yml:2]: What should be the hosts here?

File: scenarios/centos-9/hive.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+cifmw_hive_platform: "openstack"
+cifmw_hive_action: "claim_cluster"
+cifmw_hive_dry_run: false
+cifmw_hive_openstack_pool_name:
+cifmw_hive_namespace:
+claim_name:
+cifmw_hive_kubeconfig:
```
Comment (frenzyfriday) [inline scenarios/centos-9/hive.yml:8]: How should I pass these?

File: ci/playbooks/hive/claim.yml (hunk 1 above)
Comment (cjeanner) [reply ci/playbooks/hive/claim.yml:2]: I think it should be `localhost` ? i.e. the host running ansible would request the cluster. Would that make sense?

File: scenarios/centos-9/hive.yml (hunk 2 above)
Comment (cjeanner) [reply scenarios/centos-9/hive.yml:8]: are those private parameters? If so, they should be in zuul's secrets I guess.

File: zuul.d/end-to-end.yaml [hunk 3]
```diff
@@ -18,6 +18,25 @@
       - github.com/openstack-k8s-operators/install_yamls
     timeout: 5400
 
+- job:
+    name: cifmw-end-to-end-base-hive
+    nodeset: centos-9-crc-3xl
```
Comment (cjeanner) [inline zuul.d/end-to-end.yaml:23]: hmmm if we're getting a cluster via hive, we don't need to get crc... do we?

**Pull request closed without merging**